src/test/java/io/meeds/qa/ui/steps/definition/TasksStepDefinition.java

```diff
@@ -1070,7 +1070,7 @@ public void setInSearchRandomProjectField() {
   @When("^I search for the created space$")
   public void setInSearchRandomSpaceField() {
     String randomSpaceName = Serenity.sessionVariableCalled("randomSpaceName");
-    tasksSteps.setInSearchProjectField(randomSpaceName);
+    tasksSteps.setInSearchSpaceField(randomSpaceName);
```

Comment:
useless and may even make regressions, try to reuse existing scenario definition and avoid changing existing ones :
https://github.com/Meeds-io/meeds-qa-ui/blob/c01e8c4626e2a571abc351cdedaf9d51f9000622/src/test/java/io/meeds/qa/ui/steps/definition/ManageSpaceStepDefinitions.java#L344-L347